test/src/summarize_test.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dartpad.summarize_test;
+
+import 'package:dart_pad/src/summarize.dart';
+import 'package:unittest/unittest.dart';
+
+void defineTests() {
+  group('summarization', () {
```

Comment:
Let's name the group the same as what it's testing - so `summarize` or `Summarizer`.